File: week6/mario.py
from cs50 import get_int


def main():
    while True:
        # get hight
        h = get_int(" What hight is the pyramid? ")
        if h < 9 and h > 0:
            # call pyramid
            w = h
            pyramid(h, w)
            break


def pyramid(h, w):
    H = h - 1
    # till lat one
    if h > 1:
        pyramid(H, w)
    # print pyramid line
    for i in range(w - h):
        print(" ", end="")
    for i in range(h):
        print("#", end="")
    print("  ", end="")
    for i in range(h):
        print("#", end="")
    # next line
    print()


main()

File: week1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
// get hight
{
    int h;
    do
    {
        h = get_int(" What hight is the pyramid? \n");
    }
    while (1 > h || h > 8);

    int H = h;
    // print
    for (H = h; H > 0; H--)
    {
        // print spaces
        for (int w = H - 1; w > 0; w--)
        {
            printf(" ");

        }

        // print #
        for (int hash = H; hash <= h; hash++)
        {
            printf("#");
        }

        //mddle
        printf("  ");
        // second half #
        for (int hash = H; hash <= h; hash++)
        {
            printf("#");
        }

        printf("\n");
    }
}
